modules/ROOT/pages/tinydrive/getting-started.adoc

```diff
@@ -24,31 +24,31 @@ The starter projects have their own procedures to get {cloudfilemanager} up and
 [discrete]
 ==== Choose a familiar language
 
-[cols=4*]
+[cols=4*,role=center,halign=center]
```

Comment:
```suggestion
[cols="4*"]
```
`halign` is not recognised by AsciiDoctor and instead to align we should use the markup you have on the cells below instead. We also should use quotes for consistency. Either way, nice work on cleaning all this up!

Reference: https://docs.asciidoctor.org/asciidoc/latest/tables/table-ref/

Reply:
ok, sorry, probably I forgot to remove those unused attributes after the experiments to fix it